ydb/docs/en/core/dev/troubleshooting/schemas/overloaded-shards.md

```diff
@@ -0,0 +1,23 @@
+# Overloaded shards
+
+## Description
+
+[Data shards](../../../concepts/glossary.md#data-shard) in {{ ydb-short-name }} might get overloaded for the following reasons:
```

Comment:
This article discusses several related issues; it probably makes sense to adjust the common article structure for this case.

Currently:

```text
# Header

## Description
...

- Issue 1
- Issue 2

## Diagnostics

...

## Recommendations

- Issue 1
- Issue 2
```

Could be:

```text
# Header

Introduction...

- [Issue 1](#issue-1)
- [Issue 2](#issue-2)

## Issue 1 {#issue-1}

More details on the issue.

### Diagnostics

How to check whether it applies or not.

### Recommendations

...

## Issue 2 {#issue-2}

More details on the issue.

### Diagnostics

How to check whether it applies or not.

### Recommendations

...

```

Reply:
We decided on a slightly different structure here with different recommendations for each case